runtime-assets.sln

```diff
@@ -21,7 +21,7 @@ Project("{FAE04EC0-301F-11D3-BF4B-00C04F79EFBC}") = "System.Text.RegularExpressi
 EndProject
 Project("{FAE04EC0-301F-11D3-BF4B-00C04F79EFBC}") = "System.Windows.Extensions.TestData", "src\System.Windows.Extensions.TestData\System.Windows.Extensions.TestData.csproj", "{BB874A4E-3388-44EB-B4DC-40C0F38572C8}"
 EndProject
-Project("{FAE04EC0-301F-11D3-BF4B-00C04F79EFBC}") = "System.Runtime.TestData", "src\System.Runtime.TestData\System.Runtime.TestData.csproj", "{3A28B303-88E0-43B2-BFDF-9D388CC283FB}"
+Project("{FAE04EC0-301F-11D3-BF4B-00C04F79EFBC}") = "System.Runtime.TimeZoneData", "src\System.Runtime.TimeZoneData\System.Runtime.TimeZoneData.csproj", "{F806A5D2-B091-453B-B00B-13EF43973D0D}"
```

Comment:
Should System.Runtime.TestData say here and System.Runtime.TimeZoneData be added as a new Project?

Reply:
System.Runtime.TimeZoneData used to be System.Runtime.TestData